common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftTokenRequest.java

```diff
@@ -180,4 +185,12 @@ public String getMicrosoftEnrollmentId() {
     public void setMicrosoftEnrollmentId(String microsoftEnrollmentId) {
         this.mMicrosoftEnrollmentId = microsoftEnrollmentId;
     }
+
+    public String getDeviceCode() {
```

Comment:
Let's declare this value `@Nullable` (as looks to be unassigned in auth code flow)

```java
@Nullable
public String getDeviceCode() {
    // ...
}
```

Reply:
or get prepared for type annotations and place it on the String type declaration.